src/bidiMapper/domains/network/networkRequest.ts

```diff
@@ -23,9 +23,9 @@
 
 import Protocol from 'devtools-protocol';
 
-import {Deferred} from '../../../utils/deferred';
-import {IEventManager} from '../events/EventManager';
-import {Network} from '../../../protocol/protocol';
+import {Deferred} from '../../../utils/deferred.js';
```

Comment:
NIT: ideally this should be in a separate PR to reduce noise

Reply:
Agreed. This should have been reverted, it was accidentally merged. Removed.